tools/seed/seed-users-and-fleets.sh

```
@@ -0,0 +1,326 @@
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+FLEETCTL="${FLEETCTL:-./build/fleetctl}"
+$FLEETCTL config set --address "$SERVER_URL" --token "$TOKEN" 2>/dev/null
+
+# Helper: get fleet ID and exact name by partial match (to handle emoji prefixes),
+# creating the fleet if it doesn't exist. Returns "id|exact_name".
+# SECURITY: $name is interpolated into inline Python. Only pass hardcoded strings,
+# never user input, to avoid code injection.
+get_fleet_id() {
+  local name="$1"
+  local result
+  result=$(curl $CURL_FLAGS -H "$AUTH" "$API/teams" --insecure \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)